mkinitcpio

```diff
@@ -16,16 +16,16 @@ _d_hooks="$PWD/hooks:/usr/lib/initcpio/hooks:/lib/initcpio/hooks"
 _d_install="$PWD/install:/usr/lib/initcpio/install:/lib/initcpio/install"
 _d_flag_hooks=
 _d_flag_install=
-_d_firmware=({/usr,}/lib/firmware/updates {/usr,}/lib/firmware)
+_d_firmware=/usr/lib/firmware
```

Comment:
This is broken for any system that hasn't merged /lib and /usr/lib. The canonical location for firmware is /lib/firmware, not /usr/lib/firmware.

Reply:
Does Arch Linux supports unmerged /lib?